### ISSUE TITLE
Cannot compile for Cortex-M0 on GCC

### ISSUE THREAD

**Issue by SeppoTakalo** [ARMmbed/atmel-rf-driver#42]
```
Compile [  4.1%]: NanostackRfPhyAtmel.cpp
/var/folders/84/tqqdqp696j73582q658x7pkx0l_yxp/T//ccbnV0ER.s: Assembler messages:
/var/folders/84/tqqdqp696j73582q658x7pkx0l_yxp/T//ccbnV0ER.s:459: Error: instruction not supported in Thumb16 mode -- `subs r0,r0,#1'

```

CC: @kjbracey-arm

**Comment by ciarmcom** [comment]
ARM Internal Ref: ONME-2815

**Comment by SeppoTakalo** [comment]
Fixed in #43
